docs/src/main/paradox/index.md

```diff
@@ -1,273 +1,89 @@
-# Akka Distributed Workers
+# Akka Distributed Workers with Scala Guide
+ 
+Akka is a toolkit and runtime for building highly concurrent, distributed, and fault-tolerant event-driven applications on the JVM. Akka can be used with both Java and Scala.
 
-##The Goal
+This guide introduces Akka clusters by describing the Scala version of a distributed workers example. If you prefer to use Akka with Java, switch to the [Akka Distributed Workers with Java guide](http://developer.lightbend.com/guides/akka-quickstart-java/).
```

Comment:
Realised this one will be done before the Java one, so maybe best is to say "A Java version is under construction but is not yet available" and then we can update the section when it the Java one is done.